src/lib/github/prComments.ts

```diff
@@ -12,7 +12,13 @@ const formatPRComment = (snykDeltaResults: SnykDeltaOutput, commitNb: string): s
   let vulnerabilityLine = '';
   let licenseLine = '';
 
-  allIssuesToDisplay = `### ******* Vulnerabilities report for commit number ${commitNb} *******\n` 
+  if (snykDeltaResults.passIfNoBaseline)
+  {
+    allIssuesToDisplay = `### ******* INFORMATION ONLY *******\n`
```

Comment:
Add something like "Ignoring vulns if not baseline" so it's clearer what's happening.